### ISSUE TITLE
sudo inside install.sh

### ISSUE THREAD

**Issue by TermanEmil** [Yepkit/ykush#22]
There should be no `sudo` in scripts.
It gets hard to automate stuff.

I'm using ykush with a docker container and I install everything through an automated script. I get: `./install.sh: line 4: sudo: command not found`

**Comment by Dr4ik** [comment]
Good point. It is also forcing me to instal sudo inside a docker container which is strange.
It seems to be better to just cp that file instead of using ./install.sh
